src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -28,40 +28,41 @@
+            .withBlock("A").withRoom("420").withTags("owesMoney", "friends").build();
     public static final Person CARL = new PersonBuilder().withName("Carl Kurz").withPhone("95352563")
             .withEmail("heinz@example.com").withAddress("wall street")
-            .withGender("M").withMatriculationNumber("A0123456D").build();
+            .withBlock("A").withRoom("420").withGender("M").withMatriculationNumber("A0123456D").build();
     public static final Person DANIEL = new PersonBuilder().withName("Daniel Meier").withPhone("87652533")
             .withEmail("cornelia@example.com").withAddress("10th street").withGender("M")
-            .withMatriculationNumber("A0123456E").withTags("friends").build();
+            .withBlock("A").withRoom("420").withMatriculationNumber("A0123456E").withTags("friends").build();
     public static final Person ELLE = new PersonBuilder().withName("Elle Meyer").withPhone("9482224")
             .withEmail("werner@example.com").withAddress("michegan ave")
-            .withGender("F").withMatriculationNumber("A0123456F").build();
+            .withBlock("A").withRoom("420").withGender("F").withMatriculationNumber("A0123456F").build();
     public static final Person FIONA = new PersonBuilder().withName("Fiona Kunz").withPhone("9482427")
             .withEmail("lydia@example.com").withAddress("little tokyo")
-            .withMatriculationNumber("A0123456G").withGender("F").build();
+            .withBlock("A").withRoom("420").withMatriculationNumber("A0123456G").withGender("F").build();
     public static final Person GEORGE = new PersonBuilder().withName("George Best").withPhone("9482442")
             .withEmail("anna@example.com").withAddress("4th street")
-            .withMatriculationNumber("A0123456H").withGender("M").build();
+            .withBlock("A").withRoom("420").withMatriculationNumber("A0123456H").withGender("M").build();
 
 
     // Manually added
     public static final Person HOON = new PersonBuilder().withName("Hoon Meier").withPhone("8482424")
             .withEmail("stefan@example.com").withAddress("little india").withMatriculationNumber("A0123456I")
-            .withGender("M").build();
+            .withBlock("A").withRoom("420").withGender("M").build();
     public static final Person IDA = new PersonBuilder().withName("Ida Mueller").withPhone("8482131")
             .withEmail("hans@example.com").withAddress("chicago ave").withMatriculationNumber("A0123456J")
-            .withGender("F").build();
+            .withBlock("A").withRoom("420").withGender("F").build();
+
 
     // Manually added - Person's details found in {@code CommandTestUtil}
     public static final Person AMY = new PersonBuilder().withName(VALID_NAME_AMY).withPhone(VALID_PHONE_AMY)
             .withEmail(VALID_EMAIL_AMY).withAddress(VALID_ADDRESS_AMY).withTags(VALID_TAG_FRIEND)
-            .withGender(VALID_GENDER_AMY)
+            .withGender(VALID_GENDER_AMY).withBlock("A").withRoom("420")
```

Comment:
Perhaps having VALID_BLOCK and VALID_ROOM will be better for conforming with standards